hazelcast-client/src/test/java/com/hazelcast/client/replicatedmap/nearcache/ClientReplicatedMapNearCacheBasicTest.java

```diff
@@ -120,30 +120,6 @@ public void whenCacheIsFull_thenPutOnSameKeyShouldUpdateValue_withUpdateOnNearCa
     public void whenPutAllIsUsed_thenNearCacheShouldBeInvalidated_withUpdateOnNearCacheAdapter() {
     }
 
-    @Test(expected = UnsupportedOperationException.class)
-    @Override
-    public void whenReplaceIsUsed_thenNearCacheShouldBeInvalidated_withUpdateOnNearCacheAdapter() {
```

Comment:
These kind of `@Override` methods are not needed anymore. They are not indicating that an existing method is broken, just that a method is not available in this scenario.